model/feature_flags.go

```diff
@@ -85,6 +88,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.AutoTour = "none"
 	f.BoardsFeatureFlags = ""
 	f.AddMembersToChannel = "top"
+	f.GuidedChannelCreation = true
```

Comment:
Before merge, should this be set to `false`?

Reply:
good catch! done!